Doc/library/smtplib.rst

```diff
@@ -128,6 +129,11 @@ Protocol) and :rfc:`1869` (SMTP Service Extensions).
    Unix socket, LMTP generally don't support or require any authentication, but
    your mileage might vary.
 
+   .. versionchanged:: 3.9
+      The optional *timeout* parameter was added. If the *timeout* parameter is
+      set to be zero, it will raise a :class:`ValueError` to prevent the creation
+      of a non-blocking socket.
```

Comment:
"If the *timeout* parameter is set to be zero, it will raise a :class:`ValueError` to prevent the creation of a non-blocking socket." This is not technically a 3.9 change, since 3.8 doesn't have the parameter.